cdc/processor.go

```diff
@@ -290,7 +291,7 @@ func (p *oldProcessor) Run(ctx context.Context) {
 // wait blocks until all routines in processor are returned
 func (p *oldProcessor) wait() {
 	err := p.wg.Wait()
-	if err != nil && errors.Cause(err) != context.Canceled {
+	if err != nil && cerror.ErrProcessorTableCanceled.NotEqual(err) {
```

Comment:
should we also filter `context.Canceled`, otherwise the log bumps too much `context canceled` error when cdc exits normally